src/main/steps/respondent/task-list/utils.ts

```diff
@@ -91,7 +91,7 @@ export const getRespondentAllegationsOfHarmAndViolence = (userCase: CaseWithId):
 };
 
 export const getViewAllDocuments = (userCase: CaseWithId): SectionStatus => {
-  console.log(userCase?.orderCollection);
+  console.log(userCase);
```

Comment:
Please remove debug logs

Reply:
will remove the console logs, in future we may need this userCase details for checking some of the properties